packages/frontend-api/src/Resources/config/graphql-types/Query/QueryDecorator.types.yaml

```diff
@@ -308,3 +308,17 @@ QueryDecorator:
                 resolve: "@=query('complaintsQuery', args)"
                 complexity: "@=dynamicPaginationComplexity(args)"
                 description: "Returns list of complaints that can be paginated using `first`, `last`, `before` and `after` keywords"
+            isCompanyRegisteredQuery:
```

Comment:
Could you please unify it into one query with `isCustomerUserRegisteredQuery` and rename it to something better? Like `couldBeCustomerUserRegistered` or `isCustomerUSerRegistrationPossible` or anything like that